db/migrate/20221122174225_create_user.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUser < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Kindly add a `posts_counter` field of integer type as this is part of the requirements
![image](https://user-images.githubusercontent.com/81932478/203424300-8598aa0a-3fd8-4eaf-b392-598a3bab21e1.png)